src/Automation/Data/Config.cs

```diff
@@ -84,6 +93,17 @@ public Builder WithOutputDirectory(string directory)
                 return this;
             }
 
+            /// <summary>
+            /// Specifies a custom mechansm to 
```

Comment:
nit: `mechansm` -> `mechanism`, and I think the rest of this comment got cut off.

Reply:
Fixed now :)